backend-flask/README.txt

```diff
@@ -0,0 +1,25 @@
+VENV
+To install venv enter the following command:
+
+$ python3 -m venv venv
+
+This installs the venv package and names the folder "venv".
+
+
+To activate the VM, navigate to the directory where your venv folder is located.
+
+For linux users:
+$ source venv/bin/activate
+
+For windows users using cmd:
+$ venv\Scripts\activate
```

Comment:
Just a heads up that the `Activate.ps1` script didn't work for me on Powershell due to the following error.

```
.\venv\Scripts\Activate.ps1 : File .\LUG_Dungeons_and_Dragons_Webbapp\backend-flask\venv\Scripts\Activate.ps1 cannot be loaded because running scripts is disabled on this system. For more information, see
about_Execution_Policies at https:/go.microsoft.com/fwlink/?LinkID=135170.
```

However, running `activate.bat` in the regular Windows command prompt (i.e. `cmd`) worked fine.

Reply:
Might be a good idea to add in a note about setting your Powershell execution policy settings (https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/set-executionpolicy?view=powershell-5.1)

Reply:
Added a note in the Readme.